pkgs/top-level/all-packages.nix

```diff
@@ -27784,7 +27784,6 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreServices Cocoa Hypervisor;
     inherit (darwin.stubs) rez setfile;
     inherit (darwin) sigtool;
-    python = python3;
```

Comment:
this would cause `depsBuildBuild` to break, as the package set passed in all-packages.nix is not spliced.

Reply:
If true, this seems to be a huge problem for cross-compilation in nixpkgs...

Reply:
yes passing arguments to callPackage breaks splicing: https://github.com/NixOS/nixpkgs/issues/49526